app/src/main/java/com/example/ivanvelazquez/proyectointent/InfoActivity.java

```diff
@@ -0,0 +1,142 @@
+package com.example.ivanvelazquez.proyectointent;
+
+import android.content.Intent;
+import android.graphics.Color;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.text.method.ScrollingMovementMethod;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.Locale;
+import java.util.Random;
+
+import static com.example.ivanvelazquez.proyectointent.ZooAnimales.EXTRA_ANIMAL;
+
+public class InfoActivity extends AppCompatActivity implements FavoritoView.Callback {
+
+    public static final String LINK= "link de mayor informacion";
+    private Animal animal;
+    private TextView especie;
+    private TextView info;
+    private ImageView foto;
+    private TextView nombre;
+    private Button regresar;
+    private Button masInfo;
+    private String idioma = Locale.getDefault().toString();
+    private String url;
+    private FavoritoView favoritoView;
+    public static final String BACKGROUND="BACKGROUNDCOLOR";
+    public static final String BASEHOUR="BASEHOUR";
+    public int baseHour=0;
+    private int backgroundColor=0;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState){
+        super.onCreate(savedInstanceState);
+        restoreSavedInstance(savedInstanceState);
+        setContentView(R.layout.activity_info_actiity);
+        Bundle bundle = getIntent().getExtras();
+        masInfo = findViewById(R.id.btnMasInfo);
+        especie = findViewById(R.id.tvEspecie);
+        info = findViewById(R.id.tvInfo);
+        info.setMovementMethod(new ScrollingMovementMethod());
+        foto = findViewById(R.id.ivFoto);
+        regresar = findViewById(R.id.btnRegresar);
+        nombre = findViewById(R.id.tvNombre);
+        animal = (Animal) bundle.get(EXTRA_ANIMAL);
+        favoritoView = findViewById(R.id.idFavourite);
+        favoritoView.setClb(this);
+
+        especie.setText("Especie: " + animal.getEspecie());
+        info.setText(String.format("Descripcion: \n\n" + animal.getInfo() + "\n\n Atraccion y horario: " + String.format(getResources().getString(R.string.atraccion),animal.getAtraccion().getNombre(),animal.getAtraccion().getHolrario())));
```

Comment:
Mover strings al archivo `res/strings.xml` y referenciarlos desde alli. Estarias intentando poner un texto debajo del otro? Porque no usas un `LinearLayout` orientado verticalemente y adentro 2 `TextView`?

Reply:
Realizado, ademas noté un par de cosas mas para corregir, por ejemplo habian strings hardcodeados